src/components/survey/Survey.tsx

```diff
@@ -0,0 +1,90 @@
+"use client"
+import { ChangeEvent, useState, useCallback } from 'react';
+
+
+interface SurveyProps {
+  title: string;
+  lowest: string;
+  highest: string;
+}
+
+export const Survey = ({ title, lowest, highest  }: SurveyProps) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selectedRating, setSelectedRating] = useState("0");
+
+	const handleRatingClick = (event: any) => {
+		setSelectedRating(event.target.value);
+	};
+
+  const handleSubmit = (event: any) => {
+    event.preventDefault();
+    // alert(`Rating: ${selectedRating}`);
+    console.log(`Submitted rating: ${selectedRating}`);
+  };
+
+
+  return (
+    <div className="absolute bottom-0 right-0">
+       {isOpen ? (
+        <div className='flex flex-col items-end'>      
```

Comment:
There is quite a bit of repetition in regards to elements being rendered based on if this component is open or not. Could we instead render some elements/styles conditionally based on the value of `isOpen`?